sbncode/EventGenerator/MeVPrtl/config/numi_kaon_common.fcl

```diff
@@ -11,6 +11,10 @@ icarus_numi_beam2detector_rotation: [   0.921228671,    0.0226872648,   0.388359
 
 icarus_numi_beam_origin: [0, 0, 0, 4.8195e2, 79.5800e2, 794.3955e2] # cm
```

Comment:
Please see https://github.com/SBNSoftware/icaruscode/pull/230 and make sure this is consistent? (It looks like it's not). 

This and other geometry pieces look like they'd be better pulled from the geometry service to not need to worry about config in every place? Is there a reason not to do this? If not, I think it's sufficient for now to check the geometry is consistent, but we should make an issue to change this for the next major release.

Reply:
hmmm interesting it looks like there's been another update. I'll keep it in mind for the October production, not going to worry about it for now.

As far as if I can pull this from the Geometry service -- I'd love to! But I don't see how. There's now volume definition that matches the ICARUS AV (this is exactly the problem w/ "both cryostats" in GENIE). And the configuration for the beam position you linked to is XML not FCL so I don't see how I can automatically include it.

Reply:
Indeed `Geometry` (as the service) does not know where the targets are.
The transformation matrix is also not known to LArSoft: it's passed straight to `GENIEHelper`.
I don't see any easy way to get it from LArSoft; a module might be written specifically for that purpose, though.